bigquery_etl/cli/query.py

```diff
@@ -57,7 +57,7 @@ def query():
 
 @query.command(
     help="""Create a new query with name
-    <dataset>.<query_name>, for example: telemetry_derived.asn_aggregates.
+    <dataset>.<query_name>, for example: telemetry_derived.active_profiles.
```

Comment:
arbitrary, not sure whether this was supposed to have a v1 at the end

Reply:
it will add _v1 automatically if no version is specified